wordpress-mobile/WordPress-Android
# Use real device dimensions for loading featured images on discover

Parent issue #12028

This PR uses real device dimensions to calculate size of featured images. I decided to use 1/2 of the screen width to limit the size of the images (save users' data) and increase performance. Let me know what you think.

To test:
1. Enable RI FF
2. Open Stetho - network tab
3. Open Discover tab
4. Scroll to a post with a featured image
5. Verify that the request contained 0.5 of the screen width in the query parameter

PR submission checklist:
